recipes-tag/17-id-fmx-collection/meta.yaml

```diff
@@ -13,3 +13,4 @@ requirements:
   run:
     - jupyterlab
     - nodejs
+    - lmfit
```

Comment:
Please sort the dependencies alphabetically. It will make it easier to navigate through the dependencies later.